src/PanelTraits/Columns.php

```diff
@@ -263,18 +254,52 @@ public function getColumnsRelationships()
         })->toArray();
     }
 
-    // ------------
-    // TONE FUNCTIONS - UNDOCUMENTED, UNTESTED, SOME MAY BE USED
-    // ------------
-    // TODO: check them
+    /**
+     * Order the CRUD columns. If certain columns are missing from the given order array, they will be pushed to the
+     * new columns array in the original order.
+     *
+     * @param array $order An array of column names in the desired order.
+     */
+    public function orderColumns($order)
```

Comment:
Is this really ordering the columns?

To me, "ordering the columns" sounds like "sorting the columns".

If one wanted to do a German-like add words together this sounds like "putTheColumnsInTheSpecifiedOrder" or "orderColumnsAsSpecified" but those names are horrid.

In **fact** this almost sounds like `setColumnOrder` or something.

Reply:
@lloy0076, I agree, but we need to keep the method name for backwards compatibility...